.travis.yml

```diff
@@ -11,7 +11,7 @@ notifications:
 sudo: false
 script:
     - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-    - julia -e 'Pkg.clone(pwd()); Pkg.build("JSON"); Pkg.test("JSON"; coverage=true)';
+    - julia -e 'Pkg.checkout("Compat"); Pkg.clone(pwd()); Pkg.build("JSON"); Pkg.test("JSON"; coverage=true)';
```

Comment:
It turns out `tryparse` is already in `Compat`! It's just the 3-argument form (with radix as third argument) that's missing, which is currently used in parsing unicode escapes.